### ISSUE TITLE
NodeMCU for wifi connection

### ISSUE THREAD

**Issue by ReinierGielen** [REPALphilread/ReP_AL-3D-Lawn-Mower#30]
Can the NodeMCU for the wifi connection be replaced when I use an Arduino Mega with integrated Wifi ?
This would reduce a number of connections required.

**Comment by Scriptman** [comment]
I fully agree.. An integration with ESPHome for Home Assistant would also be awesome
